.github/workflows/format.yml

```diff
@@ -34,3 +34,19 @@ jobs:
           which swiftlint || brew install swiftlint
           swiftlint --fix
           git diff --exit-code || (git commit --all -m "Swift Format" && git push)
+
+  windows_cpp_format:
```

Comment:
Is it possible for this step to commit changes back like the `objc_format` job?

Reply:
Done.

Reply:
@MaikuB  If there are issues running the workflows again (I can't see any on my side due to permissions), it may be because [commits authored by GitHub Actions don't trigger workflows](https://github.com/stefanzweifel/git-auto-commit-action?tab=readme-ov-file#commits-made-by-this-action-do-not-trigger-new-workflow-runs). If there are no issues. I guess it's good to go.